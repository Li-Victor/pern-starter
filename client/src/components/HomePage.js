import React from 'react';
import { Link } from 'react-router-dom';

const HomePage = () => (
  <div>
    <p>Home page</p>
    <Link to="/libraries">Libraries page</Link>
  </div>
);

export default HomePage;
