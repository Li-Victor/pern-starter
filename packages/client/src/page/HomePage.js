import React from 'react';
import { Link } from 'react-router-dom';

const HomePage = () => (
  <React.Fragment>
    <p>Home page</p>
    <Link to='/libraries'>Libraries page</Link>
  </React.Fragment>
);

export default HomePage;
