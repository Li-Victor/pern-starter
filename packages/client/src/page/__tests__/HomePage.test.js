import React from 'react';
import { create } from 'react-test-renderer';
import { MemoryRouter } from 'react-router-dom';
import HomePage from '../HomePage';

describe('Homepage', () => {
  it('matches HomePage Snapshot', () => {
    const homePageComponent = create(
      <MemoryRouter>
        <HomePage />
      </MemoryRouter>
    );
    expect(homePageComponent.toJSON()).toMatchSnapshot();
  });
});
