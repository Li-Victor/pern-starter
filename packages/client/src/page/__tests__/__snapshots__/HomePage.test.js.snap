// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Homepage matches HomePage Snapshot 1`] = `
Array [
  <p>
    Home page
  </p>,
  <a
    href="/libraries"
    onClick={[Function]}
  >
    Libraries page
  </a>,
]
`;
